tests/integration/terminals.test.ts

```diff
@@ -0,0 +1,20 @@
+import dotenv from 'dotenv';
+import createMollieClient from '../..';
+
+/**
+ * Load the API_KEY environment variable
+ */
+dotenv.config();
+
+const mollieClient = createMollieClient({ apiKey: process.env.API_KEY });
```

Comment:
Just FYI:

Types of property 'apiKey' are incompatible.
        Type 'string | undefined' is not assignable to type 'string'.
          Type 'undefined' is not assignable to type 'string'.